openapi/components/schemas/CreditMemos/CommonCreditMemo.yaml

```
@@ -17,6 +17,10 @@ properties:
         type: array
         description: |-
           List of transactions (typically refunds, credits, and chargebacks) that are allocated to reduce the unused amount of a credit memo.
+
+            - To delete transaction allocations, send an empty transaction allocation array.
+            - To modify transaction allocations, do send a modified transaction allocation array.
+            - Not to modify transaction allocations, do not send a transaction allocation array.
```

nitpick: suggestion for readability
```suggestion
            - To delete transaction allocations, send an empty transaction allocation array in the request.
            - To modify transaction allocations, send a modified transaction allocation array in the request.
            - If you do not want to modify transaction allocations, do not send a transaction allocation array in the request.
```

---

Updated, thanks